docs/reference/search/validate.asciidoc

```diff
@@ -101,8 +145,10 @@ GET twitter/_validate/query
 --------------------------------------------------
 // TESTRESPONSE
```

Comment:
The prior `GET twitter/_validate/query` snippet needs a `// TEST[continued]` comment.